src/main/java/com/courtcanva/ccfranchise/services/OrderService.java

```diff
@@ -0,0 +1,26 @@
+package com.courtcanva.ccfranchise.services;
+
+import com.courtcanva.ccfranchise.dtos.OpenOrderResponseDto;
+import com.courtcanva.ccfranchise.exceptions.NoAvailableOrderException;
+import com.courtcanva.ccfranchise.mappers.OrderMapper;
+import com.courtcanva.ccfranchise.models.Order;
+import com.courtcanva.ccfranchise.repositories.OrderRepository;
+import lombok.RequiredArgsConstructor;
+import org.springframework.stereotype.Service;
+import java.util.List;
+
+@Service
+@RequiredArgsConstructor
+public class OrderService {
+    private final OrderRepository orderRepository;
+    private final OrderMapper orderMapper;
+
+    public List<OpenOrderResponseDto> getOpenOrdersByFranchiseeId(Long franchiseeId) {
+        List<Order> ordersFirst10 = orderRepository.findFirst10ByFranchiseeId(franchiseeId);
```

Comment:
this is to get open order and the function here is to get first 10 .... so it's not matching